db/seeds/test_data.rb

```diff
@@ -449,6 +449,8 @@
   mentor_profile.schedule = Finance::Schedule.default
 end
 
-["Capita", "Teach First", "UCL Institute of Education", "Best Practice Network", "Ambition Institute", "Education Development Trust"].each do |provider|
-  ValidTestDataGenerator::LeadProviderPopulater.call(name: provider, total_schools: 10, participants_per_school: 10)
+PaperTrail.request(enabled: false) do
+  ["Capita", "Teach First", "UCL Institute of Education", "Best Practice Network", "Ambition Institute", "Education Development Trust"].each do |provider|
+    ValidTestDataGenerator::LeadProviderPopulater.call(name: provider, total_schools: 10, participants_per_school: 10)
+  end
```

Comment:
We can also reduce it to like 1 school and 5 participants, it's not like we need 100 participants for anything.